### ISSUE TITLE
Project Creators should be able to create collections, but not to place them on the homepage

### ISSUE THREAD

**Issue by mkgold** [ManifoldScholar/manifold#1687]
## Feature Description
Adjust project collection (and associated records) authorizers to reflect the following changes:

1. Any user who can create a project can also create a project collection.
2. Manifold allows create of three types of collections: home page, instance collection, user collection
3. Users who can currently create collections (prior to implementing #1, above) can set the type of the collection.
4. Other users who can create collections are only able to create user collections.
5. Adjust the collection header to include a preview link. This will help creators get the URL of their collection.
6. Exclude user collections from the home page and from the collection list page. User collections are allowed on the collection detail route.

[List above updated by ZD on 12/10/18]

## *Why* is this feature important? *Who* does it help?
On the CUNY installation, people with the project creator role are using Manifold for OER/teaching purposes. Some of them are putting up a series of projects (where each project is a book that they are assigning) that they are using in a class. They should have the ability to create a collection of those projects so that they can share a single URL with their students and so that students can easily find the texts associated with the course.

Project creators should not be allowed to place collections on the homepage, though; I propose that that "show on homepage" option be hidden from people in this role.

## User Stories
I am a CUNY faculty member using Manifold for my class. I have created six projects, one for each book I am assigning to my students. I want to share a single URL with my students -- otherwise, they will need to find the link to each text I sent them earlier, and they also might be confused in cases where there are multiple projects with the same name (a possibility on a teaching-intensive instance like CUNY's). Having all of the projects collected in one place will ensure that everyone is reading/commenting on the correct text.

**Comment by zdavis** [comment]
Per our discussion—we're going to discuss this in person in PDX this week. Issue is on hold until then.

**Comment by jojokarlin** [comment]
Interim solution--- As long as there’s something long term

- Project creators can create collections
- Type property to homepage, instance collection, user collection
- High level users can set collections to any type
- Project creators will automatically get user collections
- Add Preview link to collection in backend -- get a url

Longterm-- Paul Page:
User Profile Pages

- Projects I’m following
- Access to annotations
- Collections I’m following

**Comment by zdavis** [comment]
I'm sorry to be the bearer of bad news, but even the interim solution is going to be too difficult to fit into 3.0. The problem is that project collections are sorted globally—that list of collections in the backend is orderable, and the order determines the order in which these collections appear in various places. As you'll recall, the original intent was for this to be a marketing feature rather than a way for professors to create ad hoc groupings of projects, which is why we included this sorting control.

The problem with what is being proposed—and I'm sorry this didn't occur to me earlier—is that updating any one collection actually updates all collections because adding a collection changes the position of all the other collections.

To address this, we'd need to change collections so that the list of collections is scoped to each available user, and so that the ordering was scoped to the collection creator. This gets weird though, because there are some collections that definitely should not be scoped to the creator (eg, everyone at UMN wants to work with the same set of collections—Terence needs to be able to manage Susan's collections).

Doing this will be a larger change than anticipated, which means that the interim solution will take too long to be a good interim solution. I think we need to turn our thinking back to a long-term, carefully modeled way for users to make their own collections of projects. I can think of some simple ways to do this—what if users could tag projects, for example, and build collections that way? This would be much simpler than trying to expose the full collection functionality to all users.

**Comment by mkgold** [comment]
so, @zdavis do you want me to create a new issue or should we continue to use this one for discussion?

**Comment by zdavis** [comment]
Let's leave this one open for now—and circle back to it for further discussion once 3.0 is done. We can refine this issue as we decide on a approach, but I think we should keep it here because the history is useful.

**Comment by zdavis** [comment]
I think that we're going to solve this issue through reading groups (via the CUNY OER work), rather than proceed in the direction outlined in this issue. Any objections to me closing this issue?

**Comment by tsmyre** [comment]
None from me.

**Comment by mkgold** [comment]
fine with me

**Comment by zdavis** [comment]
Closing because this will be part of the OER work in #2240

**Comment by zdavis** [comment]
@bot reject

**Comment by zdavis** [comment]
Thank you for taking the time to open this feature request. The Manifold team reviewed this issue during our bi-weekly meeting and the consensus is that this feature is not currently in keeping with our vision for the platform. We’re happy to continue discussing it, but for now we’re going to close the issue.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_
